components/Explore/CardLabel.tsx

```diff
@@ -1,30 +1,37 @@
 import { MaterialCommunityIcons } from "@expo/vector-icons";
 import { View, Text } from "react-native";
-import PropTypes from "prop-types";
+import { colors } from "@/theme/colors";
```

Comment:
I would solve the `CardLabelProps.font` prop a bit differently. First, add the code below to `theme/typography.ts`:

```ts
] as unknown as TypoToken[];

// Add the code below
type TextPreset = keyof (typeof typographyTokensJson)["text-styles"];

export type TextClass = `text-${TextPreset}`;
// Add the code above

export const typographyPlugin = plugin((pluginApi) => {
```

Then, update the imports in `components/Explore/CardLabel.tsx` with this:

```suggestion
import { colors } from "@/theme/colors";
import { TextClass } from "@/theme/typography";
```